src/Hooks/Variables.md

```diff
@@ -4,9 +4,17 @@ This document will provide you with the code examples for forms global variables
 
 ## Set forms to develop mode
 
-This variable will set forms to develop mode that will do the following actions:
+This variable will set forms to develop mode that will do some changes in code.
```

Comment:
_some changes_

What changes? 👀

Reply:
is is listed in the next paragraph "actions"

Reply:
I'd add then:

> This variable will set forms to develop mode, which will do the code changes listed in the 'actions' paragraph below.